src/Catalyst.Core.Lib.Tests/UnitTests/P2P/IO/Observers/DeltaHeightRequestObserverTests.cs

```diff
@@ -81,11 +81,12 @@ public async Task Can_Process_DeltaHeightRequest_Correctly()
 
             var hash = MultiHash.ComputeHash(new byte[32]);
             var cid = new Cid {Hash = hash};
-
+            
             await fakeContext.Channel.ReceivedWithAnyArgs(1)
                .WriteAndFlushAsync(new LatestDeltaHashResponse
                 {
-                    Result = new DeltaIndex {Cid = cid.ToArray().ToByteString(), Height = 0}
+                    // TODO: ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979576)